lldb/test/API/lang/swift/dwarfimporter/C/TestSwiftDWARFImporterC.py

```diff
@@ -103,7 +103,6 @@ def test_dwarf_importer_exprs(self):
     @swiftTest
     def test_negative(self):
         lldb.SBDebugger.MemoryPressureDetected()
-        self.runCmd("log enable lldb types")
```

Comment:
is this intended to be in PR?

Reply:
This is part of a separate commit "remove redundant command".
I was too lazy to create two PRs, which evidently was a mistake ;-)

Reply:
https://github.com/apple/llvm-project/pull/3640

Reply:
I didn't notice the separate commit, sorry. Trivial piggybacking is cool with me, sorry to cause a new PR.